plugins/chat/app/queries/chat/channel_memberships_query.rb

```diff
@@ -7,7 +7,7 @@ def self.call(channel:, limit: 50, offset: 0, username: nil, count_only: false)
         Chat::UserChatChannelMembership
           .joins(:user)
           .includes(:user)
-          .where(user: User.human_users.activated.not_suspended.not_staged)
+          .where(user: User.real.activated.not_suspended.not_staged.not_silenced)
```

Comment:
I think we should update the flag to reflect this fix.

Reply:
Flag?

Reply:
Sorry I meant test.

Reply:
Gotcha. Test added 👍